python/servo/testing_commands.py

```diff
@@ -30,7 +30,10 @@
 from update import updatecommandline
 import tidy
 
-here = os.path.split(__file__)[0]
+SCRIPT_PATH = os.path.split(__file__)[0]
+PROJECT_TOPLEVEL_PATH = os.path.abspath(os.path.join(SCRIPT_PATH, "..", ".."))
+WEB_PLATFORM_TESTS_PATH = os.path.join("tests", "wpt", "web-platform-tests")
+SERVO_TESTS_PATH = os.path.join("tests", "wpt", "mozilla", "tests")
```

Comment:
Can you make these absolute by utilizing `PROJECT_TOPLEVEL_PATH`? Some people (not myself) like the ability to run `mach` commands from arbitrary directories